src/ui/server/controllers/insurance/account/password-reset/index.ts

```diff
@@ -60,7 +60,7 @@ export const get = (req: Request, res: Response) =>
  */
 export const post = async (req: Request, res: Response) => {
   try {
-    const email = sanitiseValue(FIELD_ID, req.body[FIELD_ID]);
+    const email = String(sanitiseValue(FIELD_ID, req.body[FIELD_ID]));
```

Comment:
could this move below the validation error check, incase for any reason the email is null?

Reply:
Good shout, have moved